tests/chainer_tests/test_variable.py

```diff
@@ -983,358 +977,314 @@ def v():
             assert v().requires_grad is self.expected
 
 
-@testing.parameterize(
-    {'x_shape': (10,)},
-    {'x_shape': ()},
-)
-class TestVariableToCpu(unittest.TestCase):
+# TODO(hvy): Classes deriving this base class should test all backends
```

Comment:
Is this TODO?
Each subclass has two backend decorators. I don't understand which this comment is referring to.

Reply:
ideep is not tested at all. I'll remove the comment if that's okay.

Reply:
In that case it should be a TODO.
Please be more specific about what to be done.
I think it's better to put a TODO just before each line that needs to be fixed.

Reply:
That sounds good, changed accordingly.